runtime/cere/src/lib.rs

```diff
@@ -1429,7 +1447,13 @@ pub type SignedPayload = generic::SignedPayload<RuntimeCall, SignedExtra>;
 pub type CheckedExtrinsic = generic::CheckedExtrinsic<AccountId, RuntimeCall, SignedExtra>;
 
 /// Runtime migrations
-type Migrations = (migrations::Unreleased,);
+type Migrations = (
+	pallet_nomination_pools::migration::versioned_migrations::V5toV6<Runtime>,
+	pallet_nomination_pools::migration::versioned_migrations::V6ToV7<Runtime>,
+	pallet_staking::migrations::v14::MigrateToV14<Runtime>,
+	pallet_grandpa::migrations::MigrateV4ToV5<Runtime>,
+	migrations::Unreleased,
```

Comment:
You should comment the `migrations::Unreleased` here as you are not going to apply it to QANET